extensions/modules/mail/src/main/java/org/exist/xquery/modules/mail/SendEmailFunction.java

```diff
@@ -47,697 +49,592 @@
 import java.net.Socket;
 import java.util.*;
 
-//send-email specific imports
-
 /**
  * eXist Mail Module Extension SendEmailFunction
- * 
+ * <p>
  * The email sending functionality of the eXist Mail Module Extension that
- * allows email to be sent from XQuery using either SMTP or Sendmail.  
- * 
+ * allows email to be sent from XQuery using either SMTP or Sendmail.
+ *
  * @author <a href="mailto:adam@exist-db.org">Adam Retter</a>
  * @author <a href="mailto:robert.walpole@devon.gov.uk">Robert Walpole</a>
  * @author <a href="mailto:andrzej@chaeron.com">Andrzej Taramina</a>
  * @author <a href="mailto:josemariafg@gmail.com">José María Fernández</a>
- * @serial 2011-08-02
  * @version 1.6
- *
+ * @serial 2011-08-02
```

Comment:
?? why

Reply:
That was already there previously, IntelliJ just moved it down a line or two when it reformatted it